plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSessionProperties.java

```diff
@@ -155,6 +156,14 @@ public HiveSessionProperties(
                         "Enable bucket-aware execution: only use a single worker per bucket",
                         hiveConfig.isBucketExecutionEnabled(),
                         false),
+                integerProperty(
+                        PARTITION_EXECUTION_MIN_PARTITIONS,
```

Comment:
I think this might be redundant to `optimizer.table-scan-node-partitioning-min-bucket-to-task-ratio`. Could you verify?

Reply:
Good point, changed to a boolean property.